src/components/ObsDetails/ActivityItem.js

```diff
@@ -84,7 +100,10 @@ const ActivityItem = ( {
               <KebabMenu>
                 <Menu.Item
                   onPress={async ( ) => {
-                    Comment.deleteComment( item.uuid, realm, apiToken );
+                    // first delete locally
+                    Comment.deleteComment( item.uuid, realm );
+                    // then delete remotely
+                    deleteCommentMutation.mutate( item.uuid );
```

Comment:
Is the lack of error handling intentional here? That's fine, just want to understand the scope of what you were going for.

Reply:
I was trying to keep existing functionality and not add/change/break functionality. So in this case, there's no error handling because that wasn't there before, either.